iqm5q.py

```diff
@@ -0,0 +1,157 @@
+import pathlib
+
```

Comment:
import itertools
import yaml

Reply:
If you plan to use them, just commit it (together with their usage).

In more details: since we're not specifying dependencies for these files, it would be slightly more complicate
- `itertools` is standard library, so this one is fully uncontroversial (if it's useful, use it)
- `pyyaml` is not standard library, but in practice is a dependency of Qibolab, and these files are loaded by Qibolab

However, in principle we might want to drop the `pyyaml` dependency in Qibolab, at some point.
So, if you can live without, try avoiding `import yaml`.

Reply:
They are already being used on the file. For `itertools` we have the part where we set the channels and qubits and `pyyaml` is for the Zurich config. These last I will try to look for an alternative to what I was using now then.

Reply:
@stavros11 or @AleCandido  I can't commit here the changes to `iqm5q.py` because I don't have the permision to push

Reply:
Not even on the branch?

Reply:
@scarrazza can you please give permission to the tii team to push to this repository? They are the ones that will be using it mostly.

Reply:
Now as an issue #6

Reply:
Fixed, please retry now.